src/app/server.ts

```diff
@@ -138,15 +138,21 @@ if (process.env.NODE_ENV === 'production') {
             `);
 		} catch (e) {
 			const error = e as Error;
-			res.status(500).send(`<pre>${error.stack}</pre>`);
+			res.status(500).send(
+				`<pre>${
+					error.stack || `Undefined error: ${error.message}`
```

Comment:
Will the text 'Undefined error' appear anywhere? I wonder if it's safer to just show the error message 'Error: Some error', that would result in 'Undefined error: Error: Some error'